library/src/main/java/com/jmolsmobile/landscapevideocapture/VideoCaptureActivity.java

```diff
@@ -27,36 +36,38 @@
 import android.view.WindowManager;
 import android.widget.Toast;
 
-import com.jmolsmobile.landscapevideocapture.camera.CameraWrapper;
-import com.jmolsmobile.landscapevideocapture.camera.NativeCamera;
-import com.jmolsmobile.landscapevideocapture.configuration.CaptureConfiguration;
-import com.jmolsmobile.landscapevideocapture.recorder.AlreadyUsedException;
-import com.jmolsmobile.landscapevideocapture.recorder.VideoRecorder;
-import com.jmolsmobile.landscapevideocapture.recorder.VideoRecorderInterface;
-import com.jmolsmobile.landscapevideocapture.view.RecordingButtonInterface;
-import com.jmolsmobile.landscapevideocapture.view.VideoCaptureView;
-
 public class VideoCaptureActivity extends Activity implements RecordingButtonInterface, VideoRecorderInterface {
 
     public static final int RESULT_ERROR = 753245;
+
```

Comment:
General tip: please avoid committing reformatting changes, they clutter you PR and make it harder to review + they clutter your git history.

You don't have to change this here

Reply:
My bad, I did not pay enough attention. Sorry for that